docs/dev/api/insights.md

```diff
@@ -3872,22 +5529,40 @@ Return information about builds and tests run included in the build. Also, provi
      <td><p><small>| QUERY | OPTIONAL | ARRAY OF STRINGS |</small></p><p>Limit results to only those run on the specified device.</p></td>
     </tr>
   </tbody>
+  <tbody>
+    <tr>
+     <td><code>device_group</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return results only for the specified device group. Available values are: <code>private</code>, <code>public</code>.</p></td>
+    </tr>
+  </tbody>
   <tbody>
     <tr>
      <td><code>os</code></td>
      <td><p><small>| QUERY | OPTIONAL | ARRAY OF STRINGS |</small></p><p>Limit results to only those run on the specified operating systems.</p></td>
     </tr>
   </tbody>
+  <tbody>
+    <tr>
+     <td><code>source</code></td>
+       <td><p><small>| QUERY | OPTIONAL | ARRAY |</small></p><p>Return results only for tests run in virtual device cloud or real device cloud. Supported values are:</p><p>
+     <ul>
+      <li><code>rdc</code> - Real Device Cloud</li>
+      <li><code>vdc</code> - Virtual Device Cloud</li>
+    </ul></p>Default value is: <code>["vdc", "rdc"]</code>
+    </td>
+    </tr>
+  </tbody>
   <tbody>
     <tr>
      <td><code>status</code></td>
      <td><p><small>| QUERY | OPTIONAL | ARRAY of STRINGS |</small></p><p>Limit results to only those with a specified status. Supported values are:</p><p>
      <ul>
-      <li><code>complete</code></li>
-      <li><code>error</code></li>
       <li><code>passed</code></li>
+      <li><code>error</code></li>
       <li><code>failed</code></li>
-    </ul></p></td>
+      <li><code>complete</code></li>
+    </ul></p>Default value is: <code>["error", "failed", "passed", "complete"]</code>
```

Comment:
All values are default? The description of the same parameter for the [Analytics > Get Test Results](https://docs.saucelabs.com/dev/api/insights/#get-test-results) endpoint doesn't include a default.